dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -2093,47 +2096,50 @@ public void deleteWorkflow(@ApiParam(hidden = true) @Parameter(hidden = true, na
     @POST
     @Path("/github/release")
     @Timed
-    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    @Consumes(MediaType.APPLICATION_JSON)
     @UnitOfWork
     @RolesAllowed({"curator", "admin"})
     @Operation(description = "Handle a release of a repository on GitHub. Will create a workflow/service and version when necessary.", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
     @ApiOperation(value = "Handle a release of a repository on GitHub. Will create a workflow/service and version when necessary.", authorizations = {
         @Authorization(value = JWT_SECURITY_DEFINITION_NAME)})
     public void handleGitHubRelease(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user") @Auth User user,
-        @Parameter(name = "repository", description = "Repository path (ex. dockstore/dockstore-ui2)", required = true) @FormParam("repository") String repository,
-        @Parameter(name = "username", description = "Username of user on GitHub who triggered action", required = true) @FormParam("username") String username,
-        @Parameter(name = "gitReference", description = "Full git reference for a GitHub branch/tag. Ex. refs/heads/master or refs/tags/v1.0", required = true) @FormParam("gitReference") String gitReference,
-        @Parameter(name = "installationId", description = "GitHub installation ID", required = true) @FormParam("installationId") String installationId) {
+        @Parameter(name = "X-GitHub-Delivery", in = ParameterIn.HEADER, description = "A GUID to identify the GitHub webhook delivery", required = true) @HeaderParam(value = "X-GitHub-Delivery")  String deliveryId,
+        @RequestBody(description = "GitHub push event payload", required = true) PushPayload payload) {
```

Comment:
To make sure I understand... The payload from the lambda will be the whole GitHub event, which has more fields than `PushPayload` has? But the DropWizard/Jersey framework will ignore additional properties and create a `PushPayload` object? Then when we're testing locally, we just create a JSON object with those minimal fields? 

And the X-GitHub-Delivery, even though it's a header, will show up in the Swagger UI?

If so, sounds pretty easy to test locally.

Reply:
Yeah, this is what it looks like in the Swagger UI with a minimal payload representing `PushPayload`:
![image](https://github.com/dockstore/dockstore/assets/25287123/80360518-228f-4026-8cd6-32c475f02f64)

The corresponding curl looks like this and works too:
```
curl -X 'POST' \
  'http://localhost:4200/api/workflows/github/release' \
  -H 'accept: application/json' \
  -H 'X-GitHub-Delivery: 12345678' \
  -H 'Authorization: Bearer <redacted>' \
  -H 'Content-Type: application/json' \
  -d '{
  "repository": {
    "full_name": "kathy-t/ghapps-single-workflow"
  },
  "sender": {
    "login": "kathy-t"
  },
  "installation": {
    "id": 7101055
  },
  "ref": "refs/heads/master"
}'
```

I also tested sending the entire GitHub event and yeah, the framework ignores additional properties and creates a `PushPayload` object

Reply:
Confirming that this is due to us having set `objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);`